docs/reference/ilm/using-policies-rollover.asciidoc

```diff
@@ -21,6 +21,11 @@ met. Because the criteria are checked periodically, the index might grow
 slightly beyond the specified threshold. To control how often the critera are
 checked, specify the `indices.lifecycle.poll_interval` cluster setting.
 
+IMPORTANT: New indices created via rollover will not automatically inherit the
+policy used by the old index, and will not use any policy by default. Therefore,
+it is highly recommended to set up an <<indices-templates,index template>> for
+your indices which specifies the policy you wish to use for each new index.
```

Comment:
policy and rollover alias setting?

Reply:
Good point! I've updated this to mention the Rollover alias setting, and switched the link to the [ILM-with-templates](https://www.elastic.co/guide/en/elasticsearch/reference/master/_applying_a_policy_to_an_index_template.html) doc page, rather than the generic index templates page.